gno.land/pkg/integration/testdata/adduserfrom.txtar

```diff
@@ -27,8 +27,8 @@ stdout '  "BaseAccount": {'
 stdout '    "address": "g1mtmrdmqfu0aryqfl4aw65n35haw2wdjkh5p4cp",'
 stdout '    "coins": "10000000ugnot",'
 stdout '    "public_key": null,'
-stdout '    "account_number": "60",'
-stdout '    "sequence": "0"'
+stdout '    "account_number": "'${user3_account_num}'",'
```

Comment:
Tell me you got fed up with guessing account numbers and sequences without telling me you got fed up with guessing account numbers and sequences :)